opennms-config-jaxb/src/main/java/org/opennms/netmgt/config/poller/Downtime.java

```diff
@@ -47,6 +49,7 @@
 @XmlAccessorType(XmlAccessType.NONE)
 public class Downtime implements Serializable {
     private static final long serialVersionUID = -2436661386464207644L;
+    private static final List<String> s_deleteValues = Arrays.asList("always", "managed", "never");
```

Comment:
Can we use constants for these instead of strings and also update the references?

Since we only use `.contains` on this list, we could also consider using an immutable set.

Reply:
Yeah I guess I could just do a normal set of constants.  I was avoiding Enum even though it's probably more correct just to keep from messing with the objects too much.